src/utils/setParameters.js

```diff
@@ -50,7 +50,7 @@ export default function setParameters (params) {
   }
 
   // Set popup padding
-  if (params.padding) {
+  if (params.padding === 0 || params.padding) {
```

Comment:
would `params.padding !== null` work here?

Reply:
If padding happened to be `undefined` here, the condition would be true and would cause the next line to fail. I think I will just update the condition to check if padding is either a `string` or `number`. That way people who don't use typescript will not be able to pass other types.